lock-api/src/main/java/com/palantir/lock/v2/TimelockService.java

```diff
@@ -24,7 +25,7 @@
 import com.palantir.timestamp.TimestampRange;
 
 @AutoDelegate
-public interface TimelockService {
+public interface TimelockService extends Closeable {
```

Comment:
nit: I'd prefer `AutoCloseable` here. `Closeable` throws an `IOException` in its default signature, and the javadoc suggests that it tends to be used for streams. TimelockService is certainly a source/destination of data, but I wouldn't quite think of it as a stream.